docs/tutorials/adding-a-support-desk-button.md

```diff
@@ -0,0 +1,243 @@
+
+Common third-party support desk solutions that work well with ApostropheCMS include Zendesk Support Widget (a comprehensive customer service platform with ticketing and knowledge base), Intercom Chat (a conversational relationship platform with chat-based support), Freshdesk Messaging (customer support software with ticketing and automation), Help Scout Beacon (email-based customer support with a clean interface), and Crisp Live Chat (a lightweight customer messaging platform with chatbot capabilities).
+
+## Integration Approach
+
+### Adding the Support Button to the Admin Bar
+
+In ApostropheCMS, you can add a support desk button to the admin bar from the `init` method of any module.
+
+The code is straightforward, using the `self.apos.adminBar.add()` method to register the new button:
+
+<AposCodeBlock>
+
+```javascript
+export default {
+  init(self) {
+    // Add support desk button to the admin bar
+    self.apos.adminBar.add(
+      'support-desk',           // Unique identifier
+      'Get Support',            // Button label
+      false,                    // Available to all users with admin access
+      {
+        contextUtility: true,   // Places button in the right side utility area
+        icon: 'question-circle', // Icon for the button
+        tooltip: 'myProject:supportDesk' // Namespaced tooltip text for translation
+      }
+    );
+  }
+};
+```
+  <template v-slot:caption>
+    modules/asset/index.js
+  </template>
+</AposCodeBlock>
+
+In this example we have added the button to the context menu, but you may choose to add it as a main navigation item or the user drop-down menu. See the [reference documentation for other placement options](/reference/modules/admin-bar.html#add-name-label-permission-options).
+
+This single method call is all you need to add the button to the admin bar. The first parameter is a unique identifier that you'll reference in your event handler. The second parameter is the visible label (ideally using a translation string in production). The third parameter controls permissions, where `false` makes it available to all users with admin access. The final parameter is an options object that determines the button's placement and appearance.
+
+### Handling the Button Click Event
+
+Once you've added the support button to the admin bar, you need to create a browser-side component to handle clicks. This is where you'll connect your admin bar button to the third-party support desk widget's API.
+
+In your project's UI JavaScript file, add an event listener for the admin bar button click and trigger the appropriate support widget:
```

Comment:
Do not put this in `ui/src`, that code will get pushed to logged-out site visitors where it is pure overhead. Plus those calls to `apos.bus` are going to error out.

Make an apos UI app (put it in `ui/apos/apps/SupportDesk.js`, where it will automatically get invoked like `ui/src/index.js` does, including the need to export a function)

Your code shouldn't need any changes beyond putting it in the right place.

Reply:
Doh - muscle memory typing. In my tester (mostly to get the code spacing right) it is in the correct place. Thanks. I put in a note to call this location out so that people don't make the mistake I did.